app/v2/controllers/validators/RetrieveBISSValidator.scala

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package v2.controllers.validators
+
+import api.controllers.validators.RulesValidator
+import api.models.domain.TypeOfBusiness
+import api.models.errors.{MtdError, RuleTaxYearNotSupportedError}
+import cats.data.Validated
+import cats.data.Validated.Invalid
+import config.FixedConfig
+import v2.models.requestData.RetrieveBISSRequestData
+
+object RetrieveBISSValidator extends RulesValidator[RetrieveBISSRequestData] with FixedConfig {
```

Comment:
Ought to be RetrieveBISSRulesValidator, otherwise it seems like it's the "main" Retrieve validator class

Reply:
However tbh this could also probably just go into the ValidatorFactory as it's potentially just another function call there